### ISSUE TITLE
jms/serializer v3.0 support

### ISSUE THREAD

**Issue by Namoshek** [consistence/consistence-jms-serializer#13]
v3.0 of [jms/serializer](https://github.com/schmittjoh/serializer/releases) has been out for a few days now. Are there plans to support this version?

**Comment by VasekPurchart** [comment]
Hello, yes there are, I have just not had the time to look at the new version yet.

So if you would like to help, you can test it and report back if there is something not working etc, thanks.

**Comment by Namoshek** [comment]
Unit tests are still running smoothly after an update to of `jms/serializer` to `v3.0.1`:

```bash
bash-4.4# cd tests
bash-4.4# ../bin/phpunit .
PHPUnit 7.5.1 by Sebastian Bergmann and contributors.

Error:         No code coverage driver is available                                                                                                                                                         
......................................                            38 / 38 (100%)

Time: 560 ms, Memory: 10.00 MB
                                                                                                                                                         
OK (38 tests, 66 assertions)
bash-4.4# cd ..
bash-4.4# composer require jms/serializer:~3.0
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating jms/serializer (2.3.0 => 3.0.1):     Update failed (Could not delete /work/vendor/jms/serializer/tests/Fixtures: )
    Would you like to try reinstalling the package instead [yes]? yes
  - Removing jms/serializer (2.3.0)
  - Installing jms/serializer (3.0.1): Downloading (100%)
Writing lock file
Generating autoload files
bash-4.4# cd tests
bash-4.4# ../bin/phpunit .
PHPUnit 7.5.1 by Sebastian Bergmann and contributors.

Error:         No code coverage driver is available

......................................                            38 / 38 (100%)

Time: 547 ms, Memory: 10.00 MB

OK (38 tests, 66 assertions)
```

Testing it with an actual application will need a bit more time. I'll see if I find some time for it tomorrow.

**Comment by VasekPurchart** [comment]
Released in [2.2](https://github.com/consistence/consistence-jms-serializer/releases/tag/2.2).
